kedro-exercises/spaceflight/src/setup.py

```diff
@@ -52,14 +54,15 @@
         "docs": [
```

Comment:
Could you sync `docs` dependencies with `kedro` code? Some deps are redundant and some need a version update.

Reply:
I think these are in line with the requirements packaged with kedro 0.17.0 at the time, no?

Reply:
Oh, sorry I missed that it is going to be updated in the future release,